be/src/vec/data_types/data_type_decimal.h

```diff
@@ -387,9 +387,9 @@ constexpr bool IsDataTypeDecimalOrNumber =
 
 template <typename FromDataType, typename ToDataType>
     requires IsDataTypeDecimal<FromDataType> && IsDataTypeDecimal<ToDataType>
-ToDataType::FieldType convert_decimals(const typename FromDataType::FieldType& value,
-                                       UInt32 scale_from, UInt32 scale_to,
-                                       UInt8* overflow_flag = nullptr) {
+typename ToDataType::FieldType convert_decimals(const typename FromDataType::FieldType& value,
```

Comment:
why add this??

Reply:
clion compiler auto optimize , I will drop this